### ISSUE TITLE
can't find version 0.0.24 in npm

### ISSUE THREAD

**Issue by dipacs** [Kotlin/dukat#200]
In the npm global repository, still 0.0.23 is the latest and 0.0.24 can't be installed

**Comment by SUPERCILEX** [comment]
@Schahen looks like it didn't publish correctly

**Comment by Schahen** [comment]
That's true and I apologize for this. Our publication token has changed and I'm working on getting the right one. 

As soon as it will be published I will closed this.

**Comment by Schahen** [comment]
Version is published, thank you for noticing.

**Comment by ESchouten** [comment]
How can we override the default dukat version in the kotlin gradle plugin nowadays? 
Seems like
```
rootProject.kotlinNodeJs.versions.dukat.version=0.0.24
```
isn't working anymore, ```kotlinNodeJs``` is not recognized

**Comment by Schahen** [comment]
@ESchouten are you 100% sure? If yes, let's create a separate issue and investigate - I've just tried following setup and it works for me:

build.gradle
```
plugins {
    id "org.jetbrains.kotlin.js" version "1.3.60"
}

repositories {
    mavenLocal()
    mavenCentral()
}


kotlin {
    target {
        compilations.all {
            kotlinOptions.moduleKind = "commonjs"
        }
        nodejs()
    }

    sourceSets["main"].dependencies {
        implementation(kotlin("stdlib-js"))
        implementation(npm("underscore", "1.9.1"))
        implementation(npm("@types/underscore", "1.9.4"))
    }
}


rootProject.kotlinNodeJs.versions.dukat.version = "0.0.24"
```

settings.gradle
```
pluginManagement {
    repositories {
        mavenLocal()
        gradlePluginPortal()
        mavenCentral()
        jcenter()
        maven {
            url "https://dl.bintray.com/kotlin/kotlin-eap"
        }
    }
}
```

gradle.properties
```
kotlin.js.experimental.generateKotlinExternals = true
```

**Comment by ESchouten** [comment]
@Schahen Thanks!
Solved it with the following:
```
rootProject.plugins.withType<NodeJsRootPlugin> {
  (extensions[NodeJsRootExtension.EXTENSION_NAME] as NodeJsRootExtension).apply {
    versions.dukat.version = "0.0.24"
  }
}
```

**Comment by Schahen** [comment]
@ESchouten  you can do it this way but `rootProject.kotlinNodeJs.versions.dukat.version = "0.0.24"` will do ;)

**Comment by ESchouten** [comment]
@Schahen I agree your one-line statement is a lot cleaner. I can't figure out why kotlinNodeJs isn't found in my gradle rootProject, having the same problem with your demo project. So I guess the longer statement will suffice...

**Comment by lamba92** [comment]
I could not see `rootProject.kotlinNodeJs` plugin as well. Digging a little in the source code I may guess that sometimes the plugin is applied at runtime, so it's extensions is not always generated.

I also noticed that `NodeJsRootPlugin` companion object has a nice `apply()` utility function so I went in my `rootProject` for:
```kotlin
NodeJsRootPlugin.apply(project).run {
    versions.dukat.version = "0.0.24"
}
```
I guess some Gradle employees may get an heart attack reading it, but it works!
